utils/post_processing/backconvert_borealis.py

```diff
@@ -0,0 +1,229 @@
+# Copyright 2019 SuperDARN Canada, University of Saskatchewan
+# Author: Liam Graham
+
+#
+# backconvert_borealis.py
+# 2019-07-30
+# Command line tool for converting restructured and compressed
+# Borealis files back to their initial site format
+import deepdish as dd
+import numpy as np
+import os
+import sys
+import subprocess as sp
+import datetime
+import warnings
+
+def backconvert_data(data_path):
+	"""
+	Converts a restructured and compressed hdf5 borealis datafile
+	back to its original, record based format.
+	Args:
+		data_path (str): Path to the data file to be back converted
+	"""
+
+	def backconvert_pre_bfiq(data_record, data_path):
```

Comment:
for the purposes of dmap conversion and maybe other uses, it would be really handy to just be able to read the file to dictionary without the write happening in the for loop (separating the write and read functions). Could you do this? I'm thinking a function that just returns the dictionary as if you had just done a dd.io.load on the original file (group names are the record names since epoch)

Reply:
Could you also pull these functions out of the backconvert_data parent function? For me to batch these out it would be handy to only import the one I need

Reply:
Functions, separated.